src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol

```diff
@@ -592,4 +592,29 @@ contract FM_BC_Bancor_Redeeming_VirtualSupply_v1 is
             );
         }
     }
+
+    //--------------------------------------------------------------------------
+    // Overriden Internal Functions
+
+    /// @dev    Distributes issuance tokens by minting them to the receiver.
```

Comment:
Should be a `@notice`. Not developer-specific.

Reply:
Sure can change it, but I left it initially to keep the styling we had before here

Reply:
Done